src/server/chromium/crBrowser.ts

```diff
@@ -59,32 +62,11 @@ export class CRBrowser extends Browser {
     }
     browser._defaultContext = new CRBrowserContext(browser, undefined, options.persistent);
 
-    const existingTargetAttachPromises: Promise<any>[] = [];
-    // First page, background pages and their service workers in the persistent context
-    // are created automatically and may be initialized before we enable auto-attach.
```

Comment:
How can we be sure that scenario mentioned in the comment is still handled correctly?

Reply:
There are new tests for what's described in this comment.